apps/backoffice-v2/src/domains/business-reports/fetchers.ts

```diff
@@ -172,3 +172,32 @@ export const createBusinessReport = async ({
 
   return handleZodError(error, businessReport);
 };
+
+export const createBatchBusinessReport = async ({
```

Comment:
Thoughts? Trivial but might be nicer for autocomplete. It is a tradeoff.
```suggestion
export const createBusinessReportBatch = async ({
```